Doxyfile.in

```diff
@@ -387,7 +387,7 @@ IDL_PROPERTY_SUPPORT   = YES
 # all members of a group must be documented explicitly.
 # The default value is: NO.
```

Comment:
This line should also be changed.  Alternatively, to avoid such inconsistencies, perhaps we should remove all `The default value is: ...` lines.

Reply:
In my opinion the default value is still `NO` (consistent with the Doxygen manual), I changed this to `YES` to avoid repeating almost identical documentation for many statistics counters/timers.